components/table/Table.tsx

```diff
@@ -76,6 +76,7 @@ export interface TableProps<RecordType>
     filters: Record<string, Key[] | null>,
     sorter: SorterResult<RecordType> | SorterResult<RecordType>[],
     extra: TableCurrentDataSource<RecordType>,
+    action: 'paginate' | 'sort' | 'filter',
```

Comment:
Please define ts type for `action` and export it.

Reply:
Added.
Since this is my first time using typescript, hope i did it right.